### DISCUSSION TITLE
Jetpack 5.1 recently release

### DISCUSSION THREAD

**Discussion by dadaroce** [OE4T/meta-tegra#1145] [Ideas]
Hello guys,

Nvidia recently released jetpack 5.1 which will be compatible with the Orin NX. Does anyone have any idea when work will start around this new release? 

I'm not really an expert on all this, but if it's possible to help I'd like to do so if anyone can guide me.

**Comment by madisongh** [comment]
Work has already started on this [work-in-progress branch](https://github.com/OE4T/meta-tegra/tree/wip-l4t-r35.2.1).  Orin NX support isn't there yet, but I will add that once I get some of the rest of the basics out of the way.
